lib/modules/menu.js

```diff
@@ -14,17 +14,13 @@ module.hidden = true;
 
 module.beforeLoad = () => {
 	renderConsoleLink();
+	addLegacyStyling();
```

Comment:
FYI this needs to be in afterLoad (or at least go) since it's based on subreddit stylesheets, which aren't ready (perhaps not even parsed, if we're fast) in beforeLoad and which could potentially be removed by styleTweaks at some point. I'll change it though, dw about it